wikichange/src/content/content.js

```diff
@@ -78,16 +81,19 @@ const renderSlider = async (creationDate) => {
                                 .toISOString()
                                 .slice(0, 10)}" id="dateOutput" name="dateOutput" style="text-align: center;"> 
                                 <button id = "highlightButton">Highlight</button> <div id="loader"></div>
+                                <p></p>
```

Comment:
there is probably a better way to make the revision button onto another line, but will leave it to other people (i made a notion task to make the button look nicer

Reply:
Lol hahaha, <br/> is for that, but it's fine. Let @ducndh fix it